src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -593,7 +593,11 @@ def _xlateContainerInContainer(self, model, yang, configC, table):
         ccontainer = model
         #print(ccontainer['@name'])
         yang[ccontainer['@name']] = dict()
-        if not configC.get(ccontainer['@name']):
+        if configC.get(ccontainer['@name']) is None:
```

Comment:
>configC.get(ccontainer['@name']) is None [](http://example.com/codeflow?start=11&length=40)

How about
```
if ccontainer['@name'] not in configC:
```

BTW, you can extract `ccontainer['@name']` as a variable, no need to calculate many times.

And remove `#print` above.